hw/ip/otbn/data/insns.yml

```diff
@@ -82,6 +82,11 @@ encoding-schemes: enc-schemes.yml
 #  straight-line: A boolean. If true, this instruction has no effect on control
 #                 flow. Optional, default true.
 #
+#  stall-cycles: A non-negative integer. If positive, this is the
+#                number of extra cycles that the instruction takes to
+#                execute (so an instruction takes 2 cycles if
+#                stall-cycles is 1). Optional, default 0.
```

Comment:
I would find it more intuitive if we weren't specifying stall cycles, but cycles an instruction takes to execute. So stall-cycles -> cycles, with a default of 1. (In my mind, a stall is something blocking the pipeline. We don't have that, loads/stores are simply 2-cycle instructions. And there might be other instructions in the future with the same property, such as a multiplication which takes 2 cycles.)

Reply:
Good point. For some reason I was trying to come up with something that defaulted to zero, but that's a bit silly! I'll change it.